packages/core/src/app/shipping/ShippingForm.tsx

```diff
@@ -14,11 +14,12 @@ import {
     ShippingInitializeOptions,
     ShippingRequestOptions,
 } from '@bigcommerce/checkout-sdk';
-import React from 'react';
+import React, { useEffect } from 'react';
 
 import { withLanguage, WithLanguageProps } from '@bigcommerce/checkout/locale';
 
 import { usePayPalConnectAddress } from '../address/PayPalAxo';
+import { isBraintreeConnectPaymentMethod } from '../payment';
```

Comment:
I think we need to move this method to some common place. As now we are starting to create a coupling between payment and shipping folders.

Reply:
maybe we can do this in separate PR? Because it will effect other files which are not related to current changes and will be better to split this from  PRs?

cc: @serhii-tkachenko

Reply:
I will move it in PayPal Connect integration package